woocommerce/woocommerce-android
# Support tapping on links

Closes: #8939 


### Description

This PR ads behavior of opening correct links upon user clicks.

### Testing instructions

Open Privacy Settings:
- tap on the Advertising Options link: assert "Tacking & Opt Outs" page is shown
- tap on the Privacy Policy assert "Privacy Policy" page is shown
- tap on the Cookie Policy assert "Cookie Policy" page is shown

### Images/gif


https://user-images.githubusercontent.com/5845095/236901426-d4e94c87-2071-48d6-ac77-5b0e40599daa.mp4
